### PR TITLE
Add inspect command to display info of running application only

### PR DESCRIPTION
**Pull request by glours**
**- What I did**
Add an `app inspect` command to display info on running applications

**- How I did it**
Create a new command
Display information from Installation Claim for Installation, Application & Parameters section
Call the status CNAB action to display the list of running services

**- How to verify it**
TBD

**- Description for the changelog**
Add a new inspect command to display information on running applications

**- A picture of a cute animal (not mandatory but encouraged)**
image

### CODE REVIEW

File: internal/commands/inspect.go [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+	orchestrator string
+}
+
+func inspectCmd(dockerCli command.Cli) *cobra.Command {
+	var opts inspectOptions
+	cmd := &cobra.Command{
+		Use:   "inspect [APP_NAME] [OPTIONS]",
+		Short: "Shows installation and application metadata, parameters and the containers list of a running application",
+		Example: `$ docker app inspect my-installed-app
+$ docker app inspect my-app:1.0.0`,
```
Comment (silvin-lubecki) [inline internal/commands/inspect.go]: I wonder if that example is correct, shouldn't it be `docker app image inspect` ?

File: internal/commands/inspect.go [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+func inspectCmd(dockerCli command.Cli) *cobra.Command {
+	var opts inspectOptions
+	cmd := &cobra.Command{
+		Use:   "inspect [OPTIONS] RUNNING_APP",
+		Short: "Shows installation and application metadata, parameters and the containers list of a running application",
+		Example: `$ docker app inspect my-running-app
+$ docker app inspect my-running-app:1.0.0`,
+		Args:   cli.RequiresMaxArgs(1),
+		Hidden: true,
```
Comment (rumpl) [inline internal/commands/inspect.go]: Should this really be hidden?

File: internal/commands/inspect.go [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+}
+
+func inspectCmd(dockerCli command.Cli) *cobra.Command {
+	var opts inspectOptions
+	cmd := &cobra.Command{
+		Use:   "inspect [OPTIONS] RUNNING_APP",
+		Short: "Shows installation and application metadata, parameters and the containers list of a running application",
+		Example: `$ docker app inspect my-running-app
+$ docker app inspect my-running-app:1.0.0`,
+		Args:   cli.RequiresMaxArgs(1),
```
Comment (eunomie) [inline internal/commands/inspect.go]: Is it really a MaxArgs? What if we only run `docker app inspect`?
If running app is optional, it should be visible in the usage: `"inspect [OPTIONS] [RUNNING_APP]"`

File: internal/commands/inspect.go [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+	}
+
+	var statusAction bool
+	for key := range installation.Bundle.Actions {
+		if strings.HasPrefix(key, "io.cnab.status") {
+			statusAction = true
+		}
+	}
+	if !statusAction {
+		return nil
```
Comment (eunomie) [inline internal/commands/inspect.go]: shouldn't we add a message like "inspect is not available" and explaining why?

File: internal/commands/inspect.go (hunk 2 above)
Comment (glours) [reply internal/commands/inspect.go]: no, it shouldn't

File: internal/commands/inspect.go (hunk 1 above)
Comment (jcsirot) [reply internal/commands/inspect.go]: image inspect command has been move to `commands/image` package

File: cmd/cnab-run/status.go [hunk 5]
```diff
@@ -1,27 +1,205 @@
...
+		tgt = fmt.Sprintf("%d-%d", pr.tStart, pr.tEnd)
+	} else {
+		tgt = fmt.Sprintf("%d", pr.tStart)
+	}
+	return fmt.Sprintf("*:%s->%s/%s", pub, tgt, pr.protocol)
+}
+
+// Ports formats port configuration. This function is copied et adapted from docker CLI
+// see https://github.com/docker/cli/blob/d6edc912ce/cli/command/service/formatter.go#L655
+func Ports(servicePorts []swarmtypes.PortConfig) string {
```
Comment (rumpl) [inline cmd/cnab-run/status.go]: There is really no need for this function to be public

**Comment by rumpl** [comment]
Is the coverage completely wrong or does this PR really reduces the coverage?

File: cmd/cnab-run/status.go (hunk 5 above)
Comment (jcsirot) [reply cmd/cnab-run/status.go]: You're right. Fixed

File: internal/commands/inspect.go (hunk 2 above)
Comment (jcsirot) [reply internal/commands/inspect.go]: fixed

**Pull request merged**